game/src/main/resources/assets/sol_client/lang/vi_VN.lang

```diff
@@ -0,0 +1,439 @@
+sol_client.passthrough=%s
+sol_client.slider.percent=%s%%
+sol_client.slider.factor=%sx
+sol_client.slider.seconds=%ss
+
+sol_client.vertical_alignment.top=Trên
+sol_client.vertical_alignment.middle=Giữa
+sol_client.vertical_alignment.bottom=Cuối
+
+sol_client.perspective.first_person=Góc nhìn thứ nhất
+sol_client.perspective.third_person_back=Góc nhìn thứ 3 phía sau
+sol_client.perspective.third_person_front=Góc nhìn thứ 3 phía trước
+
+sol_client.easing.linear=Linear
+sol_client.easing.quad=Quad
+sol_client.easing.cubic=Cubic
+sol_client.easing.quart=Quart
+sol_client.easing.quint=Quint
+sol_client.easing.expo=Expo
+sol_client.easing.sine=Sine
+sol_client.easing.circ=Circ
+sol_client.easing.back=Back
+sol_client.easing.bounce=Bounce
+sol_client.easing.elastic=Elastic
+
+sol_client.file.edit=Chỉnh sửa tập tin...
+
+sol_client.servers=Danh sách máy chủ
+
+sol_client.key.category=Sol Client
+sol_client.key.accept_request=Chấp nhận yêu cầu
+sol_client.key.dismiss_request=Từ chối yêu cầu
+
+sol_client.mod.generic.option.enabled=Đã bật
+sol_client.mod.hud.option.scale=Độ phóng đại
+sol_client.mod.simple_hud.option.background=Nền
+sol_client.mod.simple_hud.option.backgroundColour=Màu nền
+sol_client.mod.simple_hud.option.border=Viền
+sol_client.mod.simple_hud.option.borderColour=Màu viền
+sol_client.mod.simple_hud.option.textColour=Màu chữ
+sol_client.mod.simple_hud.option.shadow=Bóng chữ
+sol_client.mod.smooth_counter_hud.option.smoothNumbers=Smooth Numbers
+
+sol_client.mod.sol_client.name=Sol Client
+sol_client.mod.sol_client.description=Các thiết lập chung của Sol Client.
+
+sol_client.mod.sol_client.option.fancyMainMenu=Màn hình chính
+sol_client.mod.sol_client.option.logoInInventory=Hiện Logo ở giao diện Túi đồ
+sol_client.mod.sol_client.mods=Mods
+sol_client.mod.sol_client.option.modsKey=Phims tắt Mods
+sol_client.mod.sol_client.edit_hud=Chỉnh sửa HUD
+sol_client.mod.sol_client.option.editHudKey=Chỉnh sửa phím tắt HUD
+sol_client.mod.sol_client.option.uiColour=Màu sắc chủ đạo
+sol_client.mod.sol_client.option.smoothUIColours=Làm mượt màu sắc chủ đạo
+sol_client.mod.sol_client.option.roundedUI=Giao diện bo tròn
+sol_client.mod.sol_client.option.buttonClicks=Âm thanh khi nhấn nút
+sol_client.mod.sol_client.option.smoothScrolling=Làm mượt cuộn trang
+sol_client.mod.sol_client.option.fancyFont=Phông chữ HD
+
+sol_client.mod.fps.name=FPS
+sol_client.mod.fps.description=Hiện FPS (Số lượng khung hình mỗi giây).
+
+sol_client.mod.coordinates.name=Toạ độ
+sol_client.mod.coordinates.description=Hiện toạ độ của bạn.
+
+sol_client.mod.coordinates.option.axisLabelColour=Axis Label Color
+sol_client.mod.coordinates.option.axisValueColour=Axis Value Color
+sol_client.mod.coordinates.option.cardinalDirection=Cardinal Direction
+sol_client.mod.coordinates.option.cardinalDirectionColour=Cardinal Direction Color
+sol_client.mod.coordinates.option.axisDirection=Axis Direction
+sol_client.mod.coordinates.option.axisDirectionColour=Axis Direction Color
```

Comment:
Still English.

Reply:
I'll look around this for a few day, words used here is a bit hard to translate all of them.

Reply:
Nevermind, decided to use another word that describe this better in Vietnamese.